src/bp/ui-shared/src/MainContainer/index.tsx

```diff
@@ -0,0 +1,27 @@
+import cx from 'classnames'
+import React, { FC } from 'react'
+import { HotKeys } from 'react-hotkeys'
+
+import style from './style.scss'
+import { MainContainerProps } from './typings'
+
+const MainContainer: FC<MainContainerProps> = props => {
+  const keyHandlers = {
+    ...(props.keyHandlers || {})
+  }
+
+  const children = React.Children.toArray(props.children)
+
+  return (
+    <HotKeys handlers={keyHandlers} keyMap={props.keyMap || {}} className={style.fullsize} focused>
```

Comment:
question : is this MainContainer used in both admin and studio ? 
question#2 I lost track a bit, are we planing on having a unified ui (admin & studio )

Reply:
No clue tbh